tests/test_cli.py

```diff
@@ -728,15 +728,16 @@ def test_shell_plus(
         (['convert', '.']),
         (['convert', '.tmuxp.yaml']),
         (['convert', '.tmuxp.yaml', '-y']),
+        (['convert', '.tmuxp.yml']),
+        (['convert', '.tmuxp.yml', '-y']),
     ],
 )
 def test_convert(cli_args, tmpdir, monkeypatch):
     # create dummy tmuxp yaml so we don't get yelled at
-    tmpdir.join('.tmuxp.yaml').write(
-        """
-session_name: hello
-    """
-    )
+    filename = '.tmuxp.yaml' if cli_args[1] == '.' else cli_args[1]
+    file_ext = filename.rsplit('.', 1)[-1]
+    assert file_ext in ['yaml', 'yml'], file_ext
```

Comment:
I think this is extracting the file extension - but I wonder if we could make it more clear?

I will try:

```suggestion
    filename = cli_args[1]
    if filename == '.':
        filename = '.tmuxp.yaml'
    file_ext = filename.rsplit('.', 1)[-1]
    assert file_ext in ['yaml', 'yml'], file_ext
```